src/Generating/ProtIntGen.h

```diff
@@ -47,7 +47,7 @@ class cProtIntGen
 	virtual ~cProtIntGen() {}
 
 	/** Generates the array of specified size into a_Values, based on given min coords. */
-	virtual void GetInts(int a_MinX, int a_MinZ, int a_SizeX, int a_SizeZ, int * a_Values) = 0;
+	virtual void GetInts(size_t a_MinX, size_t a_MinZ, size_t a_SizeX, size_t a_SizeZ, int * a_Values) = 0;
```

Comment:
This is what I wanted a review for, I'll revert a_MinX, a_MinZ but there are more index operations involving the size then there are common math operations, so keeping them as size_t reduces the number of casts.